src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,20 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+import './Todo.css';
+
+const TodoList = (props) => (
+  <ul className="todo-list">
+    {props.todos.map((todoItem, index) => (
+      <Todo
+        key={index}
+        id={todoItem.id}
+        task={todoItem.task}
+        completed={todoItem.completed}
+        toggleComplete={props.toggleComplete}
+      />
```

Comment:
using the optional map param `index` like this works great as a unique key for each todo item, but in this case each todo already has an `id` property that could be used to do the same thing. this is very handy for accounting for data that doesn't have an included `id` like that, however